src/mpyl/steps/deploy/kubernetes.py

```diff
@@ -69,26 +76,44 @@ def get_endpoint(builder: ChartBuilder) -> str:
     def execute(self, step_input: Input) -> Output:
         builder = ChartBuilder(step_input)
         chart = to_service_chart(builder)
+        project = step_input.project_execution.project
 
-        deploy_result = deploy_helm_chart(
-            self._logger, chart, step_input, builder.release_name
+        chart_path = write_helm_chart(
+            self._logger,
+            chart,
+            project.target_path,
+            step_input.run_properties,
+            builder.release_name,
         )
-        if deploy_result.success:
-            hostname = self.try_extract_hostname(chart, builder.project.name)
-            url = None
-            if hostname:
-                self._logger.info(
-                    f"Service {step_input.project_execution.name} reachable at: {hostname}"
-                )
-                endpoint = self.get_endpoint(builder)
-                url = f"{hostname}{endpoint}"
-            artifact = input_to_artifact(
-                ArtifactType.DEPLOYED_HELM_APP,
-                step_input,
-                spec=DeployedHelmAppSpec(url=f"{url}"),
+
+        if step_input.install:
+            deploy_result = deploy_helm_chart(
+                self._logger, chart_path, step_input, builder.release_name
```

Comment:
It's very hard to read from the diff, so just to confirm that I understand this correctly:
* the method `deploy_helm_chart()` used to
  * generate the yaml files by calling `write_helm_chart()`
  * install them by calling `deploy_helm_chart()`
* and with this change you now:
  * removed the call to `write_helm_chart()` from `deploy_helm_chart()`
  * call `write_helm_chart()` explicitly before
  * and then, if `install = True`, install it calling `deploy_helm_chart()`

This is it right?

If so, can you rename `deploy_helm_chart()` to `install_helm_chart()` to better describe what it does?

Reply:
Correct indeed!

But if you take the historical context of "deploy didn't always deploy" away, doesn't `deploy_helm_chart` perfectly explain what it does? Anyway, I was considering this already and I'm fine with both names so I'll rename it!

Reply:
This is now done! https://github.com/Vandebron/gh-mpyl/releases

I'll point to v0.1.0 in this pr

Reply:
My reasoning was more along the lines of _"if the only thing that `deploy_helm_chart()` does now is call `helm install`, then it probably should be called `install_helm_chart()` instead, for clarity"_. But then I realized it also upserts the namespace, so I guess we can keep the name 👍